packs/boundary/templates/boundary.nomad.tpl

```diff
@@ -0,0 +1,110 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+  datacenters = [[ .boundary.datacenters | toPrettyJson ]]
```

Comment:
I can change this post-merge, but this should probably be toStringList (new, better pattern than toPrettyJson)